docs/src/pages/main/Frameworks.tsx

```diff
@@ -18,6 +20,10 @@ export default () => {
       <div className="framework-logo-wrapper mr-2"><img src="img/icons/vue.svg" /></div>
       <span>@egjs/vue-flicking</span>
     </div>
+    <div className={`framework-logo button mr-2 ${styles["is-vue3"]}`}>
```

Comment:
![image](https://user-images.githubusercontent.com/22300/121836488-7882b400-cd0e-11eb-97c2-217e98dbb5d8.png)
![image](https://user-images.githubusercontent.com/22300/121836498-7e789500-cd0e-11eb-836c-d16eecfcfeb1.png)

I think that You have to add these two areas to the README for more people to see. :)

Reply:
I'd like to, but isn't placing the same Vue icon twice looks bad on that page?

Reply:
Um... I agree enough too. I think it's okay to just put a number like this, but what do you think?

![vue2](https://user-images.githubusercontent.com/22300/121860583-f4432780-cd33-11eb-8254-6d599f6e91b1.png)
![vue3](https://user-images.githubusercontent.com/22300/121860590-f5745480-cd33-11eb-9deb-56eea63f3dc8.png)

Reply:
I think that's too much.
We should leave more space for later frameworks like Svelte or so

Reply:
OK. Can you tell me what the appropriate guideline are for you?

Reply:
I mean, that one Vue logo can both mean Vue2 & Vue3

We can rather show a message like "if you're using Vu3, check vue3-flicking" in the vue-flicking README

Reply:
That's a good way. You can link vue2 readme and vue3 readme. If you see the trend later and there are more vue3s, you can change the main link to vue3 and I think it's a good idea.

Reply:
Good, I've updated the README